src/chainparams.h

```diff
@@ -113,4 +116,24 @@ CChainParams& Params(const std::string& chain);
  */
 void SelectParams(const std::string& chain);
 
+// bip135 begin
+/**
+ * Return the currently selected parameters. Can be changed by reading in
+ * some additional config files (e.g. CSV deployment data)
+ */
+CChainParams &ModifiableParams();
+
+/**
+ * Returns true if a deployment is considered active on a particular network
+ */
+
+bool isConfiguredDeployment(const Consensus::Params& consensusParams, const int bit);
+
+/**
+ * Dump the fork deployment parameters for the given BIP70 chain name.
```

Comment:
I cannot yet fully explain the origins of this BIP reference. It is likely I copied this snippet from another comment either in chainparamsbase.h or chainsparams.h. The earliest introduction I found was with f5ae6c98260fdd3c0ca203ce67c6467d9cdaea95 . How it relates to the actual BIP70 I'm not able to say, but the reference pervades further (rpc/mining.cpp , rpc/blockchain.cpp, qt/networkstyle.h).